test/use-swr-config.test.tsx

```diff
@@ -30,8 +25,7 @@ describe('useSWR - configs', () => {
     await screen.findByText('data: 0')
 
     // wait for the refresh interval
-    await act(() => sleep(INTERVAL * 1.5))
```

Comment:
This was misbehaving locally (flaky test, failed sometimes, passed others).

Would this be an acceptable swap? it doesn't really check anymore that we are over the INTERVAL time though, which might not be desirable.

Reply:
Sounds reasonable!